### DISCUSSION TITLE
install nextflow for all users on Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by pbpearman** [nextflow-io/nextflow#4311] [Q&A]
Hello all,
After having no success installing with instructions from [here](https://www.nextflow.io/docs/latest/getstarted.html), I installed for a single user using a bash script work around [here](https://groups.google.com/g/nextflow/c/3M7JXFtJQ2g)  However, no file named nextflow ever appeared in the local installation folder.  I can only run like `$> bash nextflow-23.04.3-all run hello`
How can I install nextflow for all users on this Ubuntu machine?
--Peter

**Comment by bentsherman** [comment]
I guess you could download the `nextflow` launcher script directly from github: https://raw.githubusercontent.com/nextflow-io/nextflow/v23.04.3/nextflow

Make it executable, put it in your $PATH, and it should work. But you will probably have to tweak some directories and environment variables to make it work with the all release for all users on the system.
